flow_action_components/GetOpportunityPartners/force-app/main/default/classes/OpportunityPartnersWrapper.cls

```diff
@@ -0,0 +1,45 @@
+/**
+* @author       : Tamar Erlich
+* @date         : October 07, 2020
+* @description  : Wrapper class for opportunity partners
+* Note          : This class is called by the GetOpportunityPartnersAction.
+* *************************************************
+*       <runTest><runTest>
+* *************************************************
+* @History
+* -------
+* VERSION | AUTHOR                | DATE               | DESCRIPTION
+* 1.0     | Tamar Erlich          | October 07, 2020   | Created
+**/
+public with sharing class OpportunityPartnersWrapper {
```

Comment:
'Wrapper' generally is used to refer to action code that calls something and then handles its outputs, and is designed specifically to enhance the thing it's wrapping. I don't use it for data structures like this. Here, I'd probably go with OpportunityPartnerSet or OpportunityPartnerDescription

Reply:
Isn't this what e are doing here? taking the query output from the action and enhancing it by adding an account name and account link?
I let the name as is because modifying it now , will require modifying the blog images and the flow as well.